--- taglib-1.9.1/taglib/mpeg/id3v1/id3v1tag.h	2013-10-08 17:50:01.000000000 +0200
+++ taglib-1.9.1-taurus/taglib/mpeg/id3v1/id3v1tag.h	2013-11-12 00:24:31.206495291 +0100
@@ -68,7 +68,7 @@
        * Decode a string from \a data.  The default implementation assumes that
        * \a data is an ISO-8859-1 (Latin1) character array.
        */
-      virtual String parse(const ByteVector &data) const;
+      String parse(const ByteVector &data) const;
 
       /*!
        * Encode a ByteVector with the data from \a s.  The default implementation
@@ -79,7 +79,7 @@
        * instead do not write an ID3v1 tag in the case that the data is not
        * ISO-8859-1.
        */
-      virtual ByteVector render(const String &s) const;
+      ByteVector render(const String &s) const;
     };
 
     //! The main class in the ID3v1 implementation
